src/main/java/edu/harvard/iq/dataverse/engine/command/impl/UpdateDataverseInputLevelsCommand.java

```diff
@@ -0,0 +1,50 @@
+package edu.harvard.iq.dataverse.engine.command.impl;
```

Comment:
Instead of adding this new command can you use the original UpdateDataverseCommand and pass in the inputlevel list?

Reply:
The `UpdateDataverseInputLevelsCommand` command wraps the `UpdateDataverseCommand` command by adding the metadata block update logic, which is not included in the `UpdateDataverseCommand`.

I explain this in more detail in the PR description section: _Special notes for your reviewer_